voxel51/cli/cli.py

```diff
@@ -274,7 +274,13 @@ def setup(parser):
     def run(parser, args):
         api = API()
 
-        data = [api.get_data_details(data_id) for data_id in args.ids]
+        response = api.batch_get_data_details(args.ids)
+
+        _log_batch_response(
+            response, success_message=None,
+            failure_message="failed to get data details")
+
+        data = [r["result"] for r in response if r["success"]]
```

Comment:
`for k in dict` returns keys, not values, so this breaks with `TypeError: string indices must be integers` when indexing `r`. Adding `itervalues` seems to work. (And turns out the key is `response`, oops)